src/lib/dom.generated.d.ts

```diff
@@ -152,6 +152,12 @@ interface BlobPropertyBag {
     type?: string;
 }
```

Comment:
Most of the changes from the RC are adding `deprecated` or switching the deprecated to multiline. Though this includes a lot of the beta feedback about the DOM changes in https://github.com/microsoft/TypeScript/issues/45047